content/docs/guides/neon-private-access.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Neon Private Access
+subtitle: Learn how to connect to your Neon database via AWS PrivateLink
+enableTableOfContents: true
+updatedOn: '2024-08-07T21:36:52.647Z'
+---
+
+<PrivatePreview />
+
+The **Neon Private Access** feature lets you securely connect to your Neon databases using [AWS PrivateLink](https://docs.aws.amazon.com/vpc/latest/privatelink/concepts.html), avoiding the open internet for improved security.
+
+## Overview
+
+In a standard setup, the client application connects to a Neon database over the open internet via the Neon proxy.
+
+With **Neon Private Access**, you can connect to your database via AWS PrivateLink instead of over the open internet. In this configuration, the private access client connects to a private instance of the Neon proxy through an [AWS endpoint service](https://docs.aws.amazon.com/vpc/latest/privatelink/configure-endpoint-service.html) provided by Neon. This endpoint service is accessible only within the same AWS region as your client and is restricted to Neon-authorized customers. With **Neon Private Access**, all traffic between the private access client application and the Neon database remains within AWS’s private network, rather than traversing the public internet.
```

Comment:
"connects to a private instance": This could convey the message that it is per customer, which is not the case. We do not want to advertise that this is not the case, but we also would not want to mislead people. I would use "dedicated instance", maybe?

Reply:
perhaps we could say something like
'connects to an instance of Neon proxy that is not accessible from the public internet'?

Reply:
Sounds good, yes!